h2o-genmodel/src/main/java/hex/genmodel/algos/targetencoder/TargetEncoderMojoModel.java

```diff
@@ -0,0 +1,123 @@
+package hex.genmodel.algos.targetencoder;
+
+import hex.genmodel.MojoModel;
+
+import java.util.*;
+
+public class TargetEncoderMojoModel extends MojoModel {
+
+  public TargetEncoderMojoModel(String[] columns, String[][] domains, String responseName) {
+    super(columns, domains, responseName);
+  }
+
+  public EncodingMaps _targetEncodingMap;
+  public Map<String, Integer> _teColumnNameToIdx = new HashMap<>();
+  public boolean _withBlending;
+  public double _inflectionPoint;
+  public double _smoothing;
+
+  // Could be passed from the model training phase so that we don't have to recompute it here. Or maybe it is fine as it should be computed only once per mojoModel
```

Comment:
the comment is misleading, this is never set from the training phase

Reply:
@honzasterba Why it is misleading? It says that during training we can compute prior mean and pass it as other parameters to mojo and MojoModel.

Reply:
@deil87 can this comment be addressed?

Reply:
@michalkurka my comment in the code could be addressed but not today I think.

Reply:
@michalkurka Done! Now prior mean is being passed from TargetEncoderModel